src/lib/support/Span.h

```diff
@@ -203,6 +203,8 @@ class Span
     pointer mDataBuf;
     size_t mDataLen;
 };
+template <class T>
+Span(T * data, size_t size) -> Span<T>;
```

Comment:
By the way, it would be nice if there were some CI that would have caught this....